checkov/terraform/checks/graph_checks/gcp/GCRContainerVulnerabilityScanningEnabled.yaml

```
@@ -0,0 +1,27 @@
+metadata:
+  id: "CKV2_GCP_11"
+  name: "GCP GCR Container Vulnerability Scanning is enabled"
+  guidelines: "This policy identifies GCP accounts where GCR Container Vulnerability Scanning is not enabled. GCR Container Analysis and other third party products allow images stored in GCR to be scanned for known vulnerabilities. Vulnerabilities in software packages can be exploited by hackers or malicious users to obtain unauthorized access to local cloud resources. It is recommended to enable vulnerability scanning for images stored in Google Container Registry."
+  category: "SECURITY"
+scope:
+  provider: "gcp"
+definition:
+  cond_type: "attribute"
+  resource_types:
+  - "google_project_services"
+  - "google_project_service"
+  attribute: "iam_instance_profile"
+  operator: "exists"
+  or:
+    - cond_type: "attribute"
+      resource_types:
+      - "google_project_services"
+      attribute: "services"
+      operator: "contains"
+      value: "containerscanning.googleapis.com"
+    - cond_type: "attribute"
+      resource_types:
+      - "google_project_service"
+      attribute: "service"
+      operator: "contains"
+      value: "containerscanning.googleapis.com"
```

```suggestion
  and:
    - cond_type: "attribute"
      resource_types:
      - "google_project_services"
      - "google_project_service"
      attribute: "iam_instance_profile"
      operator: "exists"
    - or:
        - cond_type: "attribute"
          resource_types:
          - "google_project_services"
          attribute: "services"
          operator: "contains"
          value: "containerscanning.googleapis.com"
        - cond_type: "attribute"
          resource_types:
          - "google_project_service"
          attribute: "service"
          operator: "contains"
          value: "containerscanning.googleapis.com"
```
I'm surprised this works, but we usually use the `and` operator, if it is more than one condition to be explicit

---

this is my issue my bad, first check is not part of this policy while raising PR I messed it up
removed it now

---

yeah, you are right, didn't even realize that 😄